.changelogs/nav-link-block-default.yml

```
@@ -0,0 +1,3 @@
+significance: minor
+type: changed
+entry: Use dashboard as default value for navigation link block.
```

```suggestion
significance: patch
type: fixed
links:
  - "#2465"
entry: Use dashboard as default value for navigation link block.
```
I'd say this is more a bugfix than a minor change.
Also added a link to the issue.

---

@eri-trabiccolo have just fixed the changelog entry. For the link in the editor, it should work now but requires JS to be rebuilt since the default values are set in block.json

---

@seothemes sorry, I missed the rebuild :D my bad!